### PR TITLE
Add kage manifests

### PR DESCRIPTION
**Pull request by spladug**
These are the supporting files so we can do containerized staging environments.

Once these are on the main 2X branch, any branches based on it will be stagable with kage.

:eyeglasses: @gtaylor @schwers

### CODE REVIEW

**Comment by gtaylor** [comment]
:fish:

**Comment by gtaylor** [comment]
cc'ing @ckwang8128 for reference. Also, @nramadas @phil303. @prashtx for funzies.

File: .kage/manifests/deployment.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            - name: PORT
+              value: '4444'
+            - name: HTTPS
+              value: 'false'
+
+            - name: API_HEADERS
+              value: 'false'
+            - name: API_USER_AGENT
+              value: switcharoo-reddit
+            - name: API_USER_AGENT
```
Comment (schwers) [inline .kage/manifests/deployment.yml]: Is this a duplicate from the line var above?

**Comment by schwers** [comment]
💅  Super small question about the config

File: .kage/manifests/deployment.yml (hunk 1 above)
Comment (gtaylor) [inline .kage/manifests/deployment.yml]: This is probably my fault. Indeed it is. I'm going to be nuking pretty much everything under `env` as soon as I can finish up some cluster shuffling, though.

**Comment by spladug** [comment]
:haircut:

**Comment by schwers** [comment]
:fish:

**Comment by spladug** [comment]
7a283637410c42b9b0aa45b68cf9258d6a849e7d

**Pull request closed without merging**